src/controllers/profiles.js

```diff
@@ -0,0 +1,10 @@
+const ErrorResponse = require('../utils/errorResponse');
+const User = require('../models/user');
+
+exports.getAllProfiles = async (req, res, next) => {
+  const profiles = await User.find();
+  if (!profiles) {
+    return next(new ErrorResponse('No profiles found', 404));
+  }
+  return res.status(200).json({ success: true, data: profiles });
```

Comment:
Instead of making a database call, by using authentication middleware, you can just return `req.user`.